README.md

```diff
@@ -1,2 +1,75 @@
-# RACTF Backend
-The backend API for the RACTF platform.
+<h1 align="center">RACTF Core</h1>
```

Comment:
Generally, raw HTML should be avoided as much as possible in markdown. It's fine if there's no alternative, but I don't know that the `center` is needed, and the `strong`s later on certainly aren't.

Reply:
in my opinion it's more readable than a bunch of asterisks everywhere, it's also not generally considered bad practice. the center tag is required.

Reply:
The point of markdown is that it's readable without a renderer, HTML tags don't give you that.

Reply:
I still disagree, we have to use HTML tags for centering anyway, may as well go with as it as much as possible. What's more, I'd argue that `<strong>` is far more readable by a human than some bullshit asterisks that you would only know if you know markdown